code/game/gamemodes/scoreboard.dm

```diff
@@ -158,7 +158,14 @@ GLOBAL_VAR(scoreboard) // Variable to save the scoreboard string once it's been
 		damaged_job = H.job
 		damaged_key = H.key
 
-// A recursive function to properly determine the wealthiest escapee
+// A function to determine the cash plus the account balance of the wealthiest escapee
+/datum/scoreboard/proc/get_score_person_worth(mob/living/carbon/human/H)
+	. = get_score_container_worth(H)
+
+	var/balance = H.mind.initial_account.credit_balance
```

Comment:
You probably want to do a null check on mind here.

Reply:
I had thought I shouldn't given the function that calls this already checks for a mind. Should I still add a mind check here too?

Reply:
Roger.